src/sentry/static/sentry/app/components/group/externalIssueActions.tsx

```diff
@@ -15,20 +15,20 @@ import space from 'app/styles/space';
 import {Group, GroupIntegration, IntegrationExternalIssue} from 'app/types';
 
 type Props = AsyncComponent['props'] & {
-  integration: GroupIntegration;
+  configurations: GroupIntegration[];
   group: Group;
 };
 
 type State = AsyncComponent['state'] & {
   showModal: boolean;
   action: 'create' | 'link' | null;
-  selectedIntegration: GroupIntegration;
-  issue: IntegrationExternalIssue | null;
+  selectedIntegration: GroupIntegration | null;
```

Comment:
@NisanthanNanthakumar there's definitely a preference at Sentry to use `undefined` instead of `null` so you can type this as: `selectedIntegration?: GroupIntegration` but if you prefer `null`, that's OK with me.

Reply:
@scefali For states, I think it's best to be explicit with `null`. For props, I think both work.